### DISCUSSION TITLE
Rabbit does not start

### DISCUSSION THREAD

**Discussion by Samxxx1234** [rabbitmq/rabbitmq-server#9112] [Questions]
HI, I am trying to start Rabbitmq in tanzu and in logs it says
2023-08-16 06:30:05.007142+00:00 [error] <0.630.0>                                                                                                                                            │
│ 2023-08-16 06:30:05.007142+00:00 [error] <0.630.0> BOOT FAILED                                                                                                                                │
│ 2023-08-16 06:30:05.007142+00:00 [error] <0.630.0> ===========                                                                                                                                │
│ 2023-08-16 06:30:05.007142+00:00 [error] <0.630.0> Error during startup: {error,<<"Validation failed\n\n[{&lt;&lt;\"remote-dc-replicate\"&gt;&gt;,true}] are not recognised policy settings\n │
│ 2023-08-16 06:30:05.007142+00:00 [error] <0.630.0> 

Any idea where to start searching?
Thx

**Comment by mkuratczyk** [comment]
`remote-dc-replicate` is a policy key that comes from the [Warm Standby Replication commercial plugin](https://docs.vmware.com/en/VMware-RabbitMQ-for-Kubernetes/1.5/rmq/GUID-standby-replication.html).

Seems like you have not enabled that plugin, or you are trying to use the commercial feature using an OSS build of RabbitMQ.

**Reply by Samxxx1234** [reply]
Thank you, I will check that.
